ec2removeimg

```diff
@@ -70,40 +70,40 @@ argparse.add_argument(
 # also requires one argument to be specified even if --version is specified
 # This parser behavior is true even if --version and the group are part of the
 # same subgroup
-publish_image_condition_group = argparse.add_mutually_exclusive_group()
-publish_image_condition_group.add_argument(
+remove_image_condition_group = argparse.add_mutually_exclusive_group()
+remove_image_condition_group.add_argument(
     '--image-id',
     dest='image_id',
-    help='The AMI ID of the image to be published (Optional)',
+    help='The AMI ID of the image to be removed (Optional)',
     metavar='AMI_ID'
 )
-publish_image_condition_group.add_argument(
+remove_image_condition_group.add_argument(
     '--image-name',
     dest='image_name',
-    help='The image name of the image to be published (Optional)',
+    help='The image name of the image to be removed (Optional)',
     metavar='IMAGE_NAME'
 )
 help_msg = 'An image name fragment to match the image name of the image to be '
-help_msg += 'published (Optional)'
-publish_image_condition_group.add_argument(
+help_msg += 'removed (Optional)'
+remove_image_condition_group.add_argument(
     '--image-name-frag',
     dest='image_name_frag',
     help=help_msg,
     metavar='IMAGE_NAME_FRAGMENT'
 )
 help_msg = 'A regular expression to match the image name of the image to be '
-help_msg += 'published (Optional)'
-publish_image_condition_group.add_argument(
+help_msg += 'removed (Optional)'
+remove_image_condition_group.add_argument(
     '--image-name-match',
     dest='image_name_match',
     help=help_msg,
     metavar='REGEX'
 )
 argparse.add_argument(
-    '--no-confirm',
+    '--confirm',
```

Comment:
It seems like providing `--confirm` or similar would have the opposite effect. E.g. by default there would be a confirmation prompt but a flag could be used to skip the prompt.

Reply:
Sean, since there is no confirmation with the existing tool, Robert suggested we do not change the default behavior when implementing this.  This could break anything using the tool as it would not expect this change of needing input.